packages/ember-htmlbars/tests/integration/mutable_binding_test.js

```diff
@@ -111,7 +111,7 @@ QUnit.test('using a string value through middle tier does not trigger assertion'
 
   runAppend(view);
 
-  assert.strictEqual(bottom.attrs.stuff.value, 'foo', "precond - the data propagated");
+  assert.strictEqual(bottom.attrs.stuff, 'foo', "precond - the data propagated");
```

Comment:
I see. You're right, we need to give the components mutable cells if that's what they're expecting, even thought the original source is a constant that they can't really change.